src/systemtest/java/reposense/LocalRepoSystemTest.java

```diff
@@ -0,0 +1,84 @@
+package reposense;
+
+import static reposense.util.TestUtil.loadResource;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import reposense.model.RepoConfiguration;
+import reposense.model.RepoLocation;
+import reposense.model.SupportedDomainUrlMap;
+import reposense.report.ErrorSummary;
+import reposense.util.FileUtil;
+import reposense.util.SystemTestUtil;
+import reposense.util.TestRepoCloner;
+
+public class LocalRepoSystemTest {
+
+    private static final String LOCAL_DIRECTORY_ONE_PARENT = "parent1";
+    private static final String LOCAL_DIRECTORY_TWO_PARENT = "parent2";
+    private static final String LOCAL_DIRECTORY_ONE = "parent1/test-repo";
+    private static final String LOCAL_DIRECTORY_TWO = "parent2/test-repo";
+
+    private static final String OUTPUT_DIRECTORY = "local-test";
+    private static final Path REPORT_DIRECTORY_PATH = Paths.get(OUTPUT_DIRECTORY, "reposense-report");
+
+
+    @BeforeAll
+    public static void setupLocalRepos() throws Exception {
+        TestRepoCloner.clone(new RepoConfiguration(new RepoLocation("https://github.com/reposense/testrepo-Alpha")),
+                Paths.get("."), LOCAL_DIRECTORY_ONE);
+        TestRepoCloner.clone(new RepoConfiguration(new RepoLocation("https://github.com/reposense/testrepo-Alpha")),
+                Paths.get("."), LOCAL_DIRECTORY_TWO);
+    }
+
+    @BeforeEach
+    public void setupLocalTest() throws Exception {
+        SupportedDomainUrlMap.clearAccessedSet();
+        FileUtil.deleteDirectory(OUTPUT_DIRECTORY);
+        ErrorSummary.getInstance().clearErrorSet();
+    }
+
+    @AfterEach
+    public void deleteReportDirectory() throws Exception {
+        FileUtil.deleteDirectory(OUTPUT_DIRECTORY);
+    }
+
+    @AfterAll
+    public static void deleteClonedLocalRepos() throws Exception {
+        FileUtil.deleteDirectory(LOCAL_DIRECTORY_ONE_PARENT);
+        FileUtil.deleteDirectory(LOCAL_DIRECTORY_TWO_PARENT);
+    }
+
+    @Test
+    public void testSameFinalDirectory() {
+        String cliInput = String.format("-r %s %s -s d1 -u 01/04/2022 -o local-test -t UTC+08",
```

Comment:
A minor nit: I think the `01/04/2022` should be set as a constant in this class, and the `local-test` is already declared as `OUTPUT_DIRECTORY`, that should be used instead.

Reply:
Good catch, I've fixed it